### PR TITLE
task workflow and case api update

### PR DESCRIPTION
**Pull request by atulgupta2024**
1. added outcome field to case API to store the final case outcome 
2. updated summons, bail and warrants workflows to handle the re-issue, and un-delivered status

### CODE REVIEW

File: api_specifications/case-api-0.1.0.yaml [hunk 1]
```diff
@@ -335,7 +334,7 @@ components:
           type: string
         stage:
           type: string
-        substage:
+        subStage:
```
Comment (subhashini-egov) [inline api_specifications/case-api-0.1.0.yaml:337]: @atulgupta2024 can we add descriptions for stage/substage etc.. as well?

File: api_specifications/case-api-0.1.0.yaml [hunk 2]
```diff
@@ -335,7 +334,7 @@ components:
...
           type: string
         benchId:
```
Comment (subhashini-egov) [inline api_specifications/case-api-0.1.0.yaml:339]: Description for this as well.

File: api_specifications/order-api-0.1.0.yaml [hunk 3]
```diff
@@ -235,7 +235,7 @@ components:
     Order:
       description: An order is created as an outcome of an hearing or based on an application. Order will contain a set of tasks
       type: object
-      required: [ tenantId, createdDate, orderType, status, isActive ]
+      required: [ tenantId, createdDate, orderType, isActive ]
```
Comment (subhashini-egov) [inline api_specifications/order-api-0.1.0.yaml:238]: Is createdDate in epoch time? Let's add that in the description.

**Comment by subhashini-egov** [comment]
@atulgupta2024 we need to add orderDetails attribute in the API spec and the schema for order types.

**Review by subhashini-egov — CHANGES_REQUESTED** [review]

File: api_specifications/case-api-0.1.0.yaml [hunk 4]
```diff
@@ -399,9 +394,11 @@ components:
             $ref: '#/components/schemas/AdvocateMapping'
         status:
           type: string
-          description: >-
-            The current status of the case as pulled from the Workflow service.
-            Workflow service will hold the entire history of hte case.
+          description: The current status of the case as pulled from the Workflow service. Workflow service will hold the entire history of the case.
+        outcome:
+          type: string
+          format: uuid
```
Comment (storm20198) [inline api_specifications/case-api-0.1.0.yaml:400]: Is uuid format correct? or even needed here?

File: api_specifications/case-api-0.1.0.yaml (hunk 1 above)
Comment (atulgupta2024) [reply api_specifications/case-api-0.1.0.yaml:337]: done.

File: api_specifications/case-api-0.1.0.yaml (hunk 2 above)
Comment (atulgupta2024) [reply api_specifications/case-api-0.1.0.yaml:339]: done.

File: api_specifications/order-api-0.1.0.yaml (hunk 3 above)
Comment (atulgupta2024) [reply api_specifications/order-api-0.1.0.yaml:238]: yes. added description and updated type of field

File: api_specifications/case-api-0.1.0.yaml (hunk 4 above)
Comment (atulgupta2024) [reply api_specifications/case-api-0.1.0.yaml:400]: yes. all master data is linked with a master data UUID

**Comment by atulgupta2024** [comment]
@subhashini-egov review comments addressed and updates committed

**Review by subhashini-egov — APPROVED** [review]

**Pull request merged**